src/website/src/manifest.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "Clarity Design",
+  "short_name": "Clarity",
+  "theme_color": "#0065ab",
+  "background_color": "#ffffff",
+  "display": "standalone",
+  "Scope": "/",
```

Comment:
Caps, is that an issue?

Reply:
Hmm it was generated by a tool but I will double check to see what the spec is.